### ISSUE TITLE
Totally confused with FOSUB integration.

### ISSUE THREAD

**Issue by damianprzygodzki** [api-platform/docs#242]
Hi folks, 

i am totally new in api-platform (working with sf for few years now), have been trying to figure out how to call FOSUB to life, but the result is that i am writing this issue.

After reading first two chapters ('Creating a Fully Featured API in 5 Minutes' + 'Testing and Specifying the API') i moved to the main part of docs, and started with configuration.

As config says, i am enabling fos_user integration, then i am moving to a chapter **FOSUser Bundle Integration**.

'Here’s an example of declaration of a Doctrine ORM User class' - that's great, i will use that for the start - 'You need to use serialization groups to hide some properties...' - yeah, totally understood - 'First register the following service:' - wait, what? which service, the block of code below? That's the user model class, not a service. Then i figured out, that you've meant the next chapter...

```
API Platform allows to easily add a JWT-based authentication to your API using LexikJWTAuthenticationBundle.

API Platform is fully working with LexikJWTAuthenticationBundle.

In order to install the bundle please follow their documentation.
```

Hm, no. So which service should i enable? Totally lost here.

Let's stay here. I've created model for my user. It is of course extension for FOS\UserModel\Model\User - but... 'Fatal Error: Class 'FOS\UserBundle\Model\User' not found'

So what have you meant writing this? 'API Platform Core is shipped with a bridge for FOSUserBundle. If the FOSUser bundle is enabled, this bridge will use its UserManager to create, update and delete user resources.'.

Yes, it is enabled, but which bridge? Should i just install it on my own, or what? If you are saying that it is shipped with core, but you are calling the not existing paths, hm, so where is the point.

**Comment by Simperfit** [comment]
@damianprzygodzki I think this is n[on-pushed](https://github.com/api-platform/docs/blob/master/core/fosuser-bundle.md) in the docs because here is the diff in github. We should deploy the docs to remove that.

Did you install FOSUserBundle ? because the class [exists](https://github.com/FriendsOfSymfony/FOSUserBundle/tree/master/Model)

**Comment by damianprzygodzki** [comment]
Yes, i finally figured out that - thank you - but i am creating here issue, because i believe is a part of not clear describing actions that dev should make.

I think that the process is really similar to this described in topic of JWT bundle. But form of description is different.

**Comment by Simperfit** [comment]
This is resolved by #256.
